designsystem/src/main/java/com/serranoie/app/designsystem/ui/theme/component/card/ICard.kt

```
@@ -241,13 +277,54 @@ private fun SwipeableOutlinedCard(
+                )
+                Text(
+                    text = if (isYours) "You are owed" else "You owe",
+                    style = MaterialTheme.typography.labelMedium,
+                    color = if (isYours) MaterialTheme.colorScheme.primary else MaterialTheme.colorScheme.error
+                )
+            }
+            Text(
+                text = expenseName, style = MaterialTheme.typography.titleMedium
+            )
+            Text(
+                text = "$${String.format("%.2f", amountOwed)} • $membersCount people",
```

detekt (1.23.8)

[warning] String.format("%.2f", amountOwed) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)